config/config.yaml.dist

```diff
@@ -67,6 +67,17 @@ cadence:
 #    createNonexistentDomain: false
 #    workflowExecutionRetentionPeriodInDays: 3
 
+#auditLog:
+#    enabled: true
+#
+#    driver:
+#        log:
+#            enabled: false
+#            verbosity: 1
+#
+#        database:
+#            enabled: true
+
```

Comment:
**Question**: would it be worth considering to add a config section to the developer configuration (`config.dev.yaml`) with auditlog.driver.log.enabled set to true? (audit logging by default for evaluation purposes)

Reply:
In the past we mostly used the "audit log" feature to check request bodies in the database. I don't think it's a good idea to log request bodies (to the stdout), so from a developer perspective, the database will always be the goto source.

The "regular" logs are already cluttered with lots of stuff, this would rather increase the noise in my opinion.